packages/common/src/configuration/service-configuration.ts

```diff
@@ -94,7 +107,24 @@ export class ServiceConfiguration {
                     doc: 'Timeout value after which the task has to be terminated',
                 },
             },
-
+            jobManagerConfig: {
+                activeToRunningTasksRatio: {
+                    format: Number,
+                    default: 3,
+                    // tslint:disable-next-line: max-line-length
+                    doc: `The target overload ratio of running to queued tasks. Higher ratio value will result higher queued tasks count.`,
```

Comment:
>running to queued tasks [](start = 55, length = 23)

can you change this to active to running?

Reply:
updated